vulnerable_people_form/integrations/ladcode_shielding_advice_lookup.py

```diff
@@ -0,0 +1,71 @@
+import csv
+from datetime import datetime
+import pytz
+import logging
+
+from vulnerable_people_form.form_pages.shared.logger_utils import create_log_message, log_event_names
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_NATIONAL_SHIELDING_ADVICE = 'NO'
+
+la_shielding_advice_data = {}
+
+FIELDNAMES = [
+    'ladcode',
+    'ladname',
+    'shielding_advice',
+    'start_date',
+    'start_time'
+]
+
+
+def init(shielding_advice_csv_file: str):

```

Comment:
Can you make this a class so that the state is a bit more obviously encapsulated?

Reply:
Regarding this, I thin Im inclining towards leaving it as-is. 
With confirmations in place (for new file) and RDS loading which we'll have to do, this whole module will be de-commissioned pretty quickly (as in a week or 2 tops).

Happy to take your lead though if you think we should do this.

Reply:
I really don't like that it means we have to introduce `global`s—I think every bone in my developer body tells me not to do this.

Reply:
done !